src/CattyTests/Formula/FormulaTest.swift

```diff
@@ -180,4 +180,21 @@ final class FormulaTest: XCTestCase {
         formula = Formula(formulaElement: parseTree)!
         XCTAssertFalse(formula.isSingularNumber())
     }
+
+    func testGetDisplayString() {
+        let formulaString = "1.0"
+        let formattedString = String(format: "'%@'", formulaString)
+        let formula = Formula(string: formulaString)!
+        let displayString = formula.getDisplayString()
+        XCTAssertEqual(displayString, formattedString)
+    }
+
+    func testSetDisplayStringManually() {
+        let formulaString = "1.0"
+        let formattedString = String(format: "'%@'", formulaString)
```

Comment:
To make this test more readable, please change to manually assigned `displayString` to something completely different than the formulaString which you assign during initialization. Also, please add a test for the Integer formula type and one for tested Formulas (e.g., using the add operator and a function).